networkx/algorithms/tests/test_min_edge_cover.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
My bad!
I have seen it in test_matching.py, test_distance_measures.py and many others.
And also the ``from nose.tools import *``
We should edit it too.